nautobot/docs/additional-features/jobs.md

```diff
@@ -553,57 +553,94 @@ Provision of user input (`data` values) via the CLI is not supported at this tim
+
+### 1.2.x
+
+If your test case needs to be backwards-compatible with test execution against Nautobot 1.2.x, you need to handle a couple more things manually:
+
+- Set up the `"job_logs"` database correctly for testing:
+  ```python
+  from django.conf import settings
+
+  if "job_logs" in settings.DATABASES:
+      settings.DATABASES["job_logs"] = settings.DATABASES["job_logs"].copy()
+      settings.DATABASES["job_logs"]["TEST"] = {"MIRROR": "default"}
+  ```
+- Replicate the behavior of `run_job_for_testing` manually so that your test execution most closely resembles the way the celery worker would run the test:
+  ```python
+  import uuid
+
+  from django.contrib.auth import get_user_model
+  from django.contrib.contenttypes.models import ContentType
+  
+  from nautobot.extras.context_managers import web_request_context
+  from nautobot.extras.jobs import run_job
+  from nautobot.extras.models import JobResult, Job
+
+  def run_job_for_testing(job, data=None, commit=True, username="test-user"):
+      if data is None:
+        data = {}
+      user_model = get_user_model()
+      user, _ = user_model.objects.get_or_create(username=username, is_superuser=True, password="password")
+      job_result = JobResult.objects.create(
+          name=job.class_path,
+          obj_type=ContentType.objects.get_for_model(Job),
+          user=user,
+          job_id=uuid.uuid4(),
+      )
+      with web_request_context(user=user) as request:
+          run_job(data=data, request=request, commit=commit, job_result_pk=job_result.pk)
+      return job_result
+  ```
+- Setup the `databases` field on the test class correctly, and re-create the default Statuses on `setUp` in your test classes, because `django.test.TransactionTestCase` truncates them on every `tearDown`:
+  ```python
+  from django.apps import apps
+  from django.conf import settings
+  from django.test import TransactionTestCase
+
+  from nautobot.extras.management import populate_status_choices
+
+  class MyJobTestCase(TransactionTestCase):
+    # 'job_logs' is a proxy connection to the same (default) database that's used exclusively for Job logging
+    if "job_logs" in settings.DATABASES:
+        databases = ("default", "job_logs")
 
-        if "job_logs" in settings.DATABASES:
-            settings.DATABASES["job_logs"] = settings.DATABASES["job_logs"].copy()
-            settings.DATABASES["job_logs"]["TEST"] = {"MIRROR": "default"}
+    def setUp(self):
+        super().setUp()
+        populate_status_choices(apps, None)
+  ```
```

Comment:
Not sure if this is another case of local rendering not quite matching up with ReadTheDocs, but these code blocks inside a list aren't rendering properly on RTD. Perhaps remove the list formatting, i.e. just:

````markdown
Set up the `"job_logs"` database correctly for testing:

```python
...
```

Replicate the behavior of `run_job_for_testing` manually so that your test execution most closely resembles the way the celery worker would run the test:

```python
...
```

Set up the `databases` field on the test class correctly, and re-create the default Statuses on `setUp` in your test classes, because `django.test.TransactionTestCase` truncates (deletes) them on every `tearDown`:

```python
...
```
````